test/camunda-cloud/secrets.spec.js

```diff
@@ -51,6 +51,18 @@ const valid = [
         </bpmn:extensionElements>
       </bpmn:intermediateCatchEvent>
     `))
+  },
+  {
+    name: 'property with valid value with dashes',
+    moddleElement: createModdle(createProcess(`
+      <bpmn:intermediateCatchEvent id="IntermediateCatchEvent_1">
+        <bpmn:extensionElements>
+          <zeebe:properties>
+            <zeebe:property name="bar" value="{{secrets.FOO-FOO}}" />
```

Comment:
Does `{{ secrets.FOO }}` also match? If so, let's verify that pattern works, too, through a test case.

Reply:
Thanks. Added some follow-ups, potentially making the tests more robust.